cross/arm-softfp/sources.list

```diff
@@ -0,0 +1,11 @@
+deb http://ports.ubuntu.com/ubuntu-ports/ trusty main restricted universe
+deb-src http://ports.ubuntu.com/ubuntu-ports/ trusty main restricted universe
```

Comment:
Unfortunately, Ubuntu community by Canonical had stopped the support arm-softfp since Ubuntu 12.04 (precise). In other words, the up-to-date version 14.04.XX (trusty) has being released with arm-hardfp. However, the many global vendors are still releasing arm-softfp based software platforms in the field.